src/librustc_borrowck/borrowck/mod.rs

```diff
@@ -963,6 +963,24 @@ impl<'a, 'tcx> BorrowckCtxt<'a, 'tcx> {
             .emit();
     }
 
+    fn convert_region_to_span(&self, region: ty::Region) -> Option<Span> {
```

Comment:
Maybe call this `region_end_span` or something? I feel like it's worth nothing that it only tags the *end* of the region (and that's significant to how the labels read, it'd be kind of wrong to do otherwise).